tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -84,6 +84,8 @@ Define {
 	PASSTHROUGH			"false"
 	SSP0_PCM_NAME			"Port0"
 	SSP0_CAPTURE_PCM		"Port0 2nd Capture"
+	SSP0_CAPTURE_PCM_ID		12
```

Comment:
why isn't this zero? I don't get this.

Reply:
There are 2 capture PCMs, @plbossart . Pcm0 and pcm12

Reply:
You can see the 2 PCM names just above

Reply:
I kept it consisten with the PCM name on line 86